src/models/inventoryModels/inventoryModel.ts

```
@@ -381,6 +383,26 @@ StepSequencersSchema.set("toJSON", {
+    { id: false }
+);
+kubrowPetEggSchema.set("toJSON", {
+    virtuals: true,
+    transform(_document, obj) {
+        const client = obj as IKubrowPetEggClient;
+        const db = obj as IKubrowPetEggDatabase;
+
+        client.ExpirationDate = { $date: { $numberLong: "2000000000000" } };
+        client.ItemId = toOid(db._id);
+
+        delete obj._id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)